static2/builtin/analyzer.py

```diff
@@ -15,9 +15,6 @@ def analyze_functions(static):
 # runs the recursive descent parser at address
 # how to deal with block groupings?
 def make_function_at(static, address, recurse = True):
-  if static['arch'] != "i386" and static['arch'] != "x86-64":
```

Comment:
actually, can you define what "works" mean. Because, BAP supports mips, ppc, aarch, etc (not at this PR, but will support before I'll create the final one).  So maybe it would be more pythonic not to forbid static here, but to allow it to fail if something is wrong?